Library/Homebrew/keg_relocate.rb

```diff
@@ -1,4 +1,12 @@
+require "os/mac/relocate"
```

Comment:
Is `require` smart enough/properly loaded to search `os/mac` before the top source directory? I'm about to rename these, but I think I'll still have to explicitly `require "os/mac/relocate"` at the top of `keg_relocate.rb`